Unix/installbuilder/conf/omiserver.conf

```diff
@@ -33,7 +35,7 @@ httpsport=0
 #NtlmCredsFile=/etc/opt/omi/.creds/ntlm
 
 ##
-## <NICKNAME> -- set the value of nickname.
+## <NICKNAME> -- set the value of nickname for omi related directory and file path.
 ##
 #prefix=PATH
```

Comment:
Do all these things actually work if overridden? Did you try changing to alternate paths to verify this?

I'm asking this for lines 38-45, 47-49, 55-57. I believe the non-commented out lines, 46, 50-56, and 58 are actually required because OMI defaults improperly for what installbuilder sets up.

Paul would like this to be "cleaned up". What this means, in order of priority:

1. If these are here, we need to verify they work, not just with default location, but if moved to a new location, does everything still work properly?
2. We need to decide if we want to support moving all of these things?
3. For things that we depend on (like the un-commented out lines, if we indeed still depend on them), they should probably be moved to a different section with a note to not change or system may not work properly.

This is going to take some work to figure out on a per-line basis. If you want to take this up, awesome. But do all the work that we need so we don't need to revisit this and can close out the related issue completely.

Reply:
will answer the 3 questions later today, thanks

Reply:
1. None of the directory path moved to new location will work, I have tried one by one to move to /tmp. OMI related binaries now are very depended, it doesn't work now! 
I have got following errors when move things to /tmp:
error 1: 
```
/tmp/opt/omi/bin/omiengine: Protocol_New_Listener() failed: /var/opt/omi/run/omiserver.sock2017/12/06 20:43:05 [28658,28658] ERROR: null(0): Protocol_New_Listener() failed:
```
error 2:
```
/tmp/opt/omi/bin/omiserver: Launch failed: 22017/12/06 20:43:43 [28667,28667] ERROR: null(0): Launch failed: 409012848
```
error 3:
```
2017/12/06 20:57:28 [30984,30984] FATAL: null(0): EventId=10003 Priority=FATAL failed to initialize the dispatcher: 1
/opt/omi/bin/omiserver: failed to initialize the dispatcher: 12017/12/06 20:57:28 [30984,30984] ERROR: null(0): failed to initialize the dispatcher: 768683504
```
2. We don't support moving omi things anymore now.
3. Yes, they should not be changed and should notice that in a section, I will update them, thanks.

Reply:
Forget to say that I tried to moving with files under the 3 directory and also individual file, but none of them work.
/tmp/opt
/tmp/var/opt
/tmp/etc/opt/omi
```
prefix=GNU
libdir=/tmp/opt/omi/lib
includedir=/tmp/opt/omi/include

bindir=/tmp/opt/omi/bin
agentprogram=/tmp/opt/omi/bin/omiagent
serverprogram=/tmp/opt/omi/bin/omiserver

datadir=/tmp/opt/omi/share
schemadir=/tmp/opt/omi/share/omischema
schemafile=/tmp/opt/omi/share/omischema/CIM_Schema.mof

localstatedir=/tmp/var/opt/omi

rundir=/tmp/var/opt/omi/run
pidfile=/tmp/var/opt/omi/run/omiserver.pid

logdir=/tmp/var/opt/omi/log
logfile=/tmp/var/opt/omi/log/omiserver.log

certsdir=/tmp/etc/opt/omi/ssl
pemfile=/tmp/etc/opt/omi/ssl/omi.pem
keyfile=/tmp/etc/opt/omi/ssl/omikey.pem

sysconfdir=/tmp/etc/opt/omi/conf
providerdir=/tmp/etc/opt/omi/conf
registerdir=/tmp/etc/opt/omi/conf/omiregister
configfile=/tmp/etc/opt/omi/conf/omiserver.conf
```